### DISCUSSION TITLE
upload image

### DISCUSSION THREAD

**Discussion by charismanapriandi** [prisma/prisma#7885] [Q&A] [answered]
i need to know how to upload an image or file in nextjs with prisma, and the image is saved in public folder. give me a link or something please

**Comment by ryands17** [comment, marked as answer]
@rismantrdr 👋 

I don't think there's a ready example, but you can do something like this:

1. Upload the image from the client end and get the URL in the response that has the filename
2. Save the filename in the database by calling an API from your frontend to Next.js

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
